dist/profile/files/azure.ci.jenkins.io/agents.yaml

```
@@ -0,0 +1,501 @@
+---
+jenkins:
+  clouds:
+  - azureVM:
+      azureCredentialsId: "azure-credentials"
+      cloudName: "Azure"
+      configurationStatus: "pass"
+      deploymentTimeout: 1600
+      maxVirtualMachinesLimit: 10
+      newResourceGroupName: "jenkinsinfra-agents3"
+      resourceGroupReferenceType: "new"
+      vmTemplates:
+      - agentLaunchMethod: "SSH"
+        agentWorkspace: "/home/jenkins"
+        builtInImage: "Windows Server 2016"
+        credentialsId: "jenkinsvmagents-userpass"
+        diskType: "managed"
+        doNotUseMachineIfInitFails: true
+        enableMSI: false
+        enableUAMI: false
+        ephemeralOSDisk: false
+        executeInitScriptAsRoot: true
+        existingStorageAccountName: "jnl4vmebchyhh6cmbd4dfvpa"
+        imageReference:
+          galleryImageDefinition: "jenkins-agent-ubuntu-20"
+          galleryImageVersion: "0.2.2"
```

needs to be updated in multiple places and AMIs updated
```suggestion
          galleryImageVersion: "0.2.4"
```